Test/ContentPipeline/TestProcessorContext.cs

```diff
@@ -85,6 +85,19 @@ public override void AddOutputFile(string filename)
             if (typeof(TOutput) == typeof(MaterialContent) && typeof(TInput).IsAssignableFrom(typeof(MaterialContent)))
                 return (TOutput)((object)input);
 
+            var processor = (ContentProcessor<TInput, TOutput>)typeof(ContentProcessor<TInput, TOutput>).Assembly.CreateInstance("Microsoft.Xna.Framework.Content.Pipeline.Processors."+ processorName);
+            if (processor != null) {
+                var type = processor.GetType();
+                foreach (var kvp in processorParameters)
+                {
+                    var property = type.GetProperty(kvp.Key);
+                    if (property == null)
+                        continue;
+                    property.SetValue(processor, kvp.Value);
```

Comment:
This line broke the XNA tests.
"No overload for method 'SetValue' takes 2 arguments"

Reply:
Weird. Do we not run the XNA tests on the Build system?

Reply:
Unfortunately no...

https://github.com/MonoGame/MonoGame/pull/3550#issuecomment-205644961

I should try it again and see if i can make progress on it.